pkg/pgmodel/querier.go

```diff
@@ -19,11 +19,17 @@ type Reader interface {
 // Querier queries the data using the provided query data and returns the
 // matching timeseries.
 type Querier interface {
+	// Query returns resulting timeseries for a query.
 	Query(*prompb.Query) ([]*prompb.TimeSeries, error)
+	// Select returns a series set that matches the supplied query parameters.
```

Comment:
Should we be more specific with `query parameters`, like `... matches the supplied series hints`

Reply:
I think the code shows that itself. I personally don't think that change helps with the readability,